### DISCUSSION TITLE
Why is Hacktoberfest conducted ?

### DISCUSSION THREAD

**Discussion by TKirtan** [codemistic/Non-Code#131] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. You can :
  * Post your thoughts on why is hacktoberfest conducted.
  * Share ideas on contributions.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by sajalrasto** [comment]
Hacktoberfest is all about open-source community. Motivation behind conducting it could be anywhere from exposing this to the world to accumulation of talented individual to know how they can contribute to some of the very niche technologies by committing their quality code.

For many of us, github was a place from where we could pull the code, but we never did a pull request , Hacktoberfest made me do it! Now I know it may seem very simple step but actually now millions of contributors are here learning how to contribute quality code into the community.

**Reply by TKirtan** [reply]
Great share Sajal.
